multi-vector-indexing/README.md

```diff
@@ -0,0 +1,205 @@
+    https://raw.githubusercontent.com/vespa-engine/sample-apps/master/simple-semantic-search/model/bert-base-uncased.txt
+
+$ curl -L -o model/minilm-l6-v2.onnx \
+    https://github.com/vespa-engine/sample-apps/raw/master/simple-semantic-search/model/minilm-l6-v2.onnx
+</pre>
+
+Deploy the application : 
+
+<pre data-test="exec" data-test-assert-contains="Success">
+$ vespa deploy --wait 300
+</pre>
+
+## Indexing sample Wikipedia articles
+
+Download Vespa feed client 
+
+<pre data-test="exec">
+$ FEED_CLI_REPO="https://repo1.maven.org/maven2/com/yahoo/vespa/vespa-feed-client-cli" \
+	&& FEED_CLI_VER=$(curl -Ss "${FEED_CLI_REPO}/maven-metadata.xml" | sed -n 's/.*&lt;release&gt;\(.*\)&lt;.*&gt;/\1/p') \
+	&& curl -SsLo vespa-feed-client-cli.zip ${FEED_CLI_REPO}/${FEED_CLI_VER}/vespa-feed-client-cli-${FEED_CLI_VER}-zip.zip \
+	&& unzip -o vespa-feed-client-cli.zip
+</pre>
+
+Index the Wikipedia articles. This embeds all the paragraphs using the native embedding model, which
+is computionally expensive for CPU. For production use cases, use Vespa Cloud with GPU instances and
+[autoscaling](https://cloud.vespa.ai/en/autoscaling) enabled. 
+
+<pre data-test="exec">
+$ zstdcat ext/articles.jsonl.zst | \
+./vespa-feed-client-cli/vespa-feed-client \
+--stdin --endpoint http://localhost:8080
+</pre>
+
+## Query and ranking examples
+We demonstrate using `vespa cli`, use `-v` to see the curl equivalent using HTTP api.  
+
+### Simple retrieve all articles with undefined ranking:
+
+<pre data-test="exec" data-test-assert-contains='"totalCount": 8'>
+vespa query 'yql=select * from articles where true' \
+'ranking=unranked'
+</pre>
+
+### Traditional keyword search with BM25 ranking on the article level:
+<pre data-test="exec" data-test-assert-contains='24-hour clock'>
+vespa query 'yql=select * from articles where userQuery()' \
+'query=24' \
+'ranking=bm25'
+</pre>
+
+Notice the `relevance`, which is assigned by the rank-profile. Also note
+that keywords are highlighted in the `paragraphs` field. 
+
+### Semantic vector search on the paragraph level. 
+
+<pre data-test="exec" data-test-assert-contains='24-hour clock'>
+vespa query 'yql=select * from articles where {targetHits:1}nearestNeighbor(paragraph_embeddings,q)' \
+'input.query(q)=embed(what does 24 mean in the context of railways)' \
+'ranking=semantic'
+</pre>
+The closest (best semantic matching) paragraph has index 4.
+```json
+"matchfeatures": {
+ "closest(paragraph_embeddings)": {"4": 1.0}
+ }
+```
+This index corresponds to the following paragraph:
+```
+"In railway timetables 24:00 means the \"end\" of the day. For example, a train due to arrive at a station during the last minute of a day arrives at 24:00; but trains which depart during the first minute of the day go at 00:00."
+```
+The [tensor presentation format](search/query-profiles/default.xml) is overridden in
+this sample application to shorten down the output. 
+
+### Hybrid search and ranking
```

Comment:
This section is great, but I would like a bit more on the multi-vector aspect of it. I assume we will add links here to doc/guides for more information. so this is good for now. Later there should be a heading with "multiple vectors" or the likes

Reply:
Trying to keep this short and sweet.